build.gradle

```diff
@@ -21,12 +22,12 @@ repositories {
 
 dependencies {
     implementation "ru.vyarus:gradle-quality-plugin:4.7.0"
-    implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.9'
-    implementation ("com.diffplug.spotless:spotless-plugin-gradle:6.6.0") {
+    implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.12'
+    implementation ("com.diffplug.spotless:spotless-plugin-gradle:6.11.0") {
```

Comment:
Spotless plugin versions after 6.6.0 caused problems the last time we tried to update. I tried pulling in a snapshot version based on this branch, and I'm seeing the same issue. 

The problem appears when using Coppuccino in a project and updating dependencies in that project. Doing so adds a bunch of old dependencies to the lockfile. For example:

```
org.eclipse.jdt:org.eclipse.jdt.core:3.25.0=spotless-1911100560
org.eclipse.platform:org.eclipse.core.commands:3.9.800=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.core.contenttype:3.7.900=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.core.jobs:3.10.1100=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.core.resources:3.14.0=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.core.runtime:3.20.100=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.equinox.app:1.5.100=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.equinox.common:3.14.100=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.equinox.preferences:3.8.200=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.equinox.registry:3.10.100=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.jface.text:3.17.0=spotless-1452293708
org.eclipse.platform:org.eclipse.jface:3.22.100=spotless-1452293708
org.eclipse.platform:org.eclipse.osgi:3.16.200=spotless-1452293708,spotless-1911100560
org.eclipse.platform:org.eclipse.text:3.11.0=spotless-1452293708,spotless-1911100560
```

Dependencies like the ones shown above introduce 3 CVEs:

- CVE-2019-9843
- CVE-2020-8908
- CVE-2021-41033

This seems to be related to a [known issue](https://github.com/diffplug/spotless/issues/1215) which is still not resolved in version 6.11.0. I think we should stick with version 6.6.0 for now and add a note stating that the dependency should not be updated until the issue is resolved.

Reply:
Ok. I'll move it back.